app/view/templates/homeopt.php

```diff
@@ -22,12 +22,10 @@
                     <legend>Sort</legend>
                     <div>
                         <select name="sortby" id="sortby">
-                            <?php
-                            foreach (Wcms\Opt::SORTBYLIST as $col) {
-                                $name = Wcms\Model::METADATAS_NAMES[$col];
-                                echo '<option value="' . $col . '" ' . ($opt->sortby() == $col ? "selected" : "") . '>' . $name . '</option>';
-                            }
-                            ?>
+                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) :
+                                $name = Wcms\Model::METADATAS_NAMES[$col]; ?>
+                                <option value="<?= $col ?>" <?= ($opt->sortby() == $col ? "selected" : "") ?> ><?= $name ?></option>
+                            <?php endforeach ?>
```

Comment:
Or simply (as $name is not used anywhere else):

```suggestion
                            <?php foreach (Wcms\Opt::SORTBYLIST as $col) : ?>
                                <option value="<?= $col ?>" <?= ($opt->sortby() == $col ? "selected" : "") ?> >
                                    <?= Wcms\Model::METADATAS_NAMES[$col] ?>
                                </option>
                            <?php endforeach ?>
```